<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>InitialScreen</class>
 <widget class="QWidget" name="InitialScreen">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>initial_screen</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="styleSheet">
      <string notr="true">font: 18px;
</string>
     </property>
     <property name="currentIndex">
      <number>3</number>
     </property>
     <widget class="QWidget" name="tabFlashcards">
      <attribute name="title">
       <string>Flashcards</string>
      </attribute>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QWidget" name="widget" native="true"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabTaskTracker">
      <attribute name="title">
       <string>Tarefas</string>
      </attribute>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QWidget" name="widget_2" native="true"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabPomodoro">
      <attribute name="title">
       <string>Pomodoro</string>
      </attribute>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QWidget" name="widget_3" native="true"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabSeeProgress">
      <attribute name="title">
       <string>Ver Progresso</string>
      </attribute>
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <item>
        <widget class="QWidget" name="widget_4" native="true"/>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
